sklearn/linear_model/tests/test_logistic.py

```diff
@@ -542,12 +542,12 @@ def test_logistic_regressioncv_class_weights():
     X, y = make_classification(n_samples=20, n_features=20, n_informative=10,
                                n_classes=3, random_state=0)
 
-    # Test the liblinear fails when class_weight of type dict is
-    # provided, when it is multiclass. However it can handle
-    # binary problems.
+    msg = ("In LogisticRegressionCV the liblinear solver cannot handle "
```

Comment:
But it does support ``class_weight``, right? It's odd to support ``balanced`` but not dicts.